packages/grid/x-grid/test/XGrid.test.tsx

```diff
@@ -1,7 +1,20 @@
 import * as React from 'react';
-import { createClientRender, act } from 'test/utils';
+import { createClientRender, act, fireEvent } from 'test/utils';
 import { expect } from 'chai';
 import { XGrid } from '@material-ui/x-grid';
+import { useData } from 'packages/storybook/src/hooks/useData.ts';
```

Comment:
@dtassone please use absolute imports anytime possible (basically if it's not inside the source of the component). This makes it much easier to navigate the codebase, as well a refractory it.